.github/workflows/sync-main-with-releases.yml

```
@@ -50,49 +49,62 @@ jobs:
             -X POST \
             -H 'Content-Type: application/json' \
-            "${{ secrets.TOOLING_TEAM_CHAT_API }}" \
+            "${{ secrets.EDITOR_TEAM_CHAT_API }}" \
             -d "$body"
+
+  notify-failure:
+    needs: [sync-main]
+    if: ${{ always() && contains(needs.*.result, 'failure') && github.repository == 'wso2/mi-vscode' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)